Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,130 @@
+# Glossary
```

Comment:
Can we add a definition for Koor Storage Distribution (KSD)? Anything else Koor-specific? Maybe Extended Ceph Exporter?

Reply:
Sure, sounds good. How do we want to define KSD ? I think we can leave Extended Ceph exporter out as it's monitoring project right rather than a terminology or a concept.

Reply:
@gauravsitlani I think mainly as a that KSD is a product based on Rook Ceph that makes running and maintaining Ceph storage on Kubernetes easier. What do you think?

I don't think we need to mention the extended-ceph-exporter as it is not mentioned/ used in the Koor docs (yet?), but it makes sense from a "promoting" the project perspective. In the future we should probably mention the extended-ceph-exporter in the monitoring documentation as well.